stl/inc/utility

```diff
@@ -423,26 +419,21 @@ template <class _Tuple>
 struct tuple_size;
 
 template <class _Tuple, class = void>
-struct _Tuple_size_sfinae { // selected when tuple_size<_Tuple>::value isn't well-formed
-};
+struct _Tuple_size_sfinae {}; // selected when tuple_size<_Tuple>::value isn't well-formed
 
 template <class _Tuple>
 struct _Tuple_size_sfinae<_Tuple, void_t<decltype(tuple_size<_Tuple>::value)>>
-    : integral_constant<size_t, tuple_size<_Tuple>::value> { // selected when tuple_size<_Tuple>::value is well-formed
-};
+    : integral_constant<size_t, tuple_size<_Tuple>::value> {}; // selected when tuple_size<_Tuple>::value is well-formed
 
 template <class _Tuple>
-struct tuple_size<const _Tuple> : _Tuple_size_sfinae<_Tuple> { // size of const tuple
-};
+struct tuple_size<const _Tuple> : _Tuple_size_sfinae<_Tuple> {}; // size of const tuple
 
 template <class _Tuple>
-struct _CXX20_DEPRECATE_VOLATILE tuple_size<volatile _Tuple> : _Tuple_size_sfinae<_Tuple> { // size of volatile tuple
-};
+struct _CXX20_DEPRECATE_VOLATILE tuple_size<volatile _Tuple> : _Tuple_size_sfinae<_Tuple> {}; // size of volatile tuple
 
 template <class _Tuple>
-struct _CXX20_DEPRECATE_VOLATILE tuple_size<const volatile _Tuple>
-    : _Tuple_size_sfinae<_Tuple> { // size of const volatile tuple
-};
+struct _CXX20_DEPRECATE_VOLATILE tuple_size<const volatile _Tuple> : _Tuple_size_sfinae<_Tuple> {
+}; // size of const volatile tuple

```

Comment:
```suggestion
struct _CXX20_DEPRECATE_VOLATILE tuple_size<const volatile _Tuple> : _Tuple_size_sfinae<_Tuple> {
    // size of const volatile tuple
};
```

Reply:
I'm going to shorten these to `// ignore cv` (as we don't have room for `-qualifiers`), although we could also drop them entirely.